.github/workflows/benchmark-command.yml

```diff
@@ -40,14 +40,14 @@ on:
           - 8h
           - 16h
           - 24h
+      folds:
+        description: 'Number of folds to run'
+        required: true
```

Comment:
My understanding is that the num of folds are not required and we can run benchmark fine without it

Reply:
Number of folds are pre-defined in the constraints.yaml - https://github.com/autogluon/autogluon/pull/3608/files#diff-b973100f40432a8431c41887e66d9a98912e3864c3e22a7cfdab54fa69259847.
This functionality is added so that we could control it if we wanted, currently I have set it as required. I can change it to False and default values will be used. 
However, the number of folds run in the PR must be equal to number of folds run on master

Reply:
required should be False imo.

Reply:
Yup will change this!